ibl_pipeline/analyses/behavior.py

```diff
@@ -6,6 +6,69 @@
 schema = dj.schema(dj.config.get('database.prefix', '') + 'ibl_analyses_behavior')
 
 
+@schema
+class Contrasts(dj.Computed):
+    definition = """
+    -> behavior.TrialSet
+    ---
+    contrasts:                      blob    # contrasts used in this session, negative when on the left
+    has_no_contrast:                bool    # have trials with no contrast
+    has_left:                       bool    # have trials with contrast on the left
+    has_right:                      bool    # have trials with contrast on the right
+    contrasts_left=null:            blob    # contrasts shown on the left
+    contrasts_right=null:           blob    # contrasts shown on the right
+    n_trials_stim:                  blob    # number of trials for each contrast
+    n_trials_stim_left=null:        blob    # number of trials when the stimulus is shown on the left
+    n_trials_stim_right=null:       blob    # number of trials when the stimulus is shown on the right
+    n_trials_stim_no_contrast=null: int     # number of trials when no stimulus is shown 
+    """
+
+    def make(self, key):
+        print(key)
+        key_con = key.copy()
```

Comment:
Is there a better name than `key_con`.  It's not used as a key and `con` is not clear. Let's follow PEP8